app/src/main/java/org/fossasia/openevent/Receivers/NotificationReceiver.java

```diff
@@ -0,0 +1,23 @@
+package org.fossasia.openevent.Receivers;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import org.fossasia.openevent.Services.BookmarkAlarmService;
+import org.fossasia.openevent.utils.IntentStrings;
+
+/**
+ * Created by Manan Wason on 21/08/15.
+ */
+public class NotificationReceiver extends BroadcastReceiver {
```

Comment:
More like the Alarm Receiver, fix the name, `NotifcationAlarmReceiver` would be a good place to start.